differential-expression/README.md

```diff
@@ -1,25 +1,100 @@
-# refine.bio Examples: Differential Expression  
-This directory contains example workflows of how to use identify differentially
-expressed genes in data from [refine.bio](https://www.refine.bio).
+# Refine.bio Example Workflow: Differential expression
 
-***
-### Differential Expression Analyses in R
+[This notebook](https://alexslemonade.github.io/refinebio-examples/differential-expression/gene_DE.nb.html)
+takes data and metadata from refine.bio and identifies differentially expressed genes.
+
+## Requirements and usage
+
+This module requires you to install the following software to run examples yourself:
+
+* [**R**](https://cran.r-project.org/)
+* [**Bioconductor**](https://bioconductor.org/install/)
+* [**tidyverse**](https://www.tidyverse.org/)
+* [**RStudio**](https://www.rstudio.com/products/RStudio/) for working with [R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html).
+
+These requirements can be installed by following the instructions at the links above.
+The example R Notebooks are designed to check if additional required packages are installed and will install them if they are not.
+
+#### RStudio
+
+We have prepared [a quick guide to RStudio](https://github.com/AlexsLemonade/training-modules/blob/master/intro_to_R_tidyverse/00-rstudio_guide.md) as part of our training content that you may find helpful if you're getting started with RStudio for the first time.
+
+Note that the first time you open RStudio, you should select a CRAN mirror.
+You can do so by clicking `Tools` > `Global Options` > `Packages` and selecting a CRAN mirror near you with the `Change` button.
+
+#### Interacting with R Notebooks
+
+You can open an R Notebook by opening the `.Rmd` file in RStudio.
+Note that working with R Notebooks requires certain R packages, but RStudio should prompt you to download them the first time you open one.
+This will allow you to modify and run the R code chunks.
+Chunks that have already been included in an example can be run by clicking the green play button in the top right corner of the chunk or by using **Ctrl + Shift + Enter** (**Cmd + Shift + Enter** on a Mac).
+See [this guide using to R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html#using-notebooks) for more information about inserting and executing code chunks.
 
-[This notebook](https://github.com/AlexsLemonade/refinebio-examples/blob/master/differential-expression/gene_DE.Rmd)
-illustrates how to identify genes that are differentially expressed in a dataset
-using the `limma` package.
-It also creates a volcano plot and Q-Q plot in order to assess the performance
-of the linear model.    
+#### Using your own data
```

Comment:
Could we add this here:
>A statement about downloading in the README (#62 (comment)):
To run the examples yourself, you will need to clone or download this repository < https://github.com/AlexsLemonade/refinebio-examples >

Reply:
Added that